packages/ui/src/components/Table.svelte

```diff
@@ -57,67 +60,86 @@
         {}
       )
     }))
+
+  const widthSty = (f: { width?: number }) =>
+    f.width !== undefined ? `min-width: ${f.width}px; width: ${f.width}px` : ''
+
+  let rerender = false
+  export function reset () {
```

Comment:
Could we avoid this kind of dirty tricks? It is unclear about it should be called outside for proper functionality.

Please check for data field updates.